### PR TITLE
Allow inserting multiple environment variables

### PR DESCRIPTION
**Pull request by jelly**
Podman users might have file or a list of environment variables they
want to add to a new container. As adding them one by one is quite
cumbersome handle inserting FOO=bar and multiple lines of such formatted
env variables.

Closes: #816

### CODE REVIEW

File: src/ImageRunModal.jsx [hunk 1]
```diff
@@ -119,18 +120,37 @@ const PublishPort = ({ id, item, onChange, idx, removeitem, itemCount }) =>
         </Grid>
     );
 
+const handleEnvValue = (key, value, idx, onChange, additem, itemCount) => {
+    // Allow the input of KEY=VALUE seperated value pairs for bulk import
+    if (value.includes('=')) {
+        const parts = value.trim().split(" ");
```
Comment (marusak) [inline src/ImageRunModal.jsx:126]: Why splitting on space? Shouldn;t we split on new line? (although it works)

File: src/ImageRunModal.jsx [hunk 2]
```diff
@@ -119,18 +120,37 @@ const PublishPort = ({ id, item, onChange, idx, removeitem, itemCount }) =>
...
+        let index = idx;
+        for (const part of parts) {
+            const [envKey, envVar] = part.split('=', 2);
```
Comment (marusak) [inline src/ImageRunModal.jsx:129]: Can you please check if both values are defined? If for example you would have empty line in between items or spaces after item, it leads to many empty items. We should skip those.
Screenshot from 2021-12-10 12-52-43
Screenshot from 2021-12-10 12-52-06

File: test/check-application [hunk 3]
```diff
@@ -1360,8 +1360,8 @@ class TestApplication(testlib.MachineCase):
         b.set_input_text('#run-image-dialog-env-2-value', 'GRAPE')
         b.click('#run-image-dialog-env-2-btn-close')
         b.click('.env-form .btn-add')
-        b.set_input_text('#run-image-dialog-env-2-key', 'RHUBARB')
-        b.set_input_text('#run-image-dialog-env-2-value', 'STRAWBERRY')
+        # Test inputting an key=var entry
+        b.set_input_text('#run-image-dialog-env-2-value', "RHUBARB=STRAWBERRY", value_check=False)
```
Comment (marusak) [inline test/check-application:1364]: Can you please test it with more values at the same time? To test the "bulk" part as well.

**Review by marusak — CHANGES_REQUESTED** [review]
Looks great! One question, one improvement and making tests happy :)

File: src/ImageRunModal.jsx (hunk 1 above)
Comment (jelly) [reply src/ImageRunModal.jsx:126]: It seems that the browser for a single line input converts newlines to spaces.

File: test/check-application (hunk 3 above)
Comment (jelly) [reply test/check-application]: Sure, done.

**Review by marusak — APPROVED** [review]
Thank you!

**Pull request merged**